util/test_base.h

```diff
@@ -41,16 +43,18 @@ class test_base {
   /** called before this test is executed
    *  @param log for emitting test notes and results
    */
-  virtual bool setup(class logger &) {
-    // stub
-    return true;
-  }
+  virtual bool setup(class logger &) { return true; }
 
-  /** execute this test
+  /** method that will be overriden in test file
```

Comment:
```suggestion
  /** member function that will be overridden in test file
```

Reply:
I've applied this comment in this commit: 4a4aa32e58ee565190fd5c4ff444133660b1ea7b